internal/distro/distro_test_common/distro_test_common.go

```diff
@@ -97,10 +97,10 @@ func TestDistro_Manifest(t *testing.T, pipelinePath string, prefix string, regis
 					imageType,
 					*tt.ComposeRequest.Blueprint,
 					distro.ImageOptions{
-						OSTree: ostree.RequestParams{
-							URL:    "foo",
-							Ref:    "bar",
-							Parent: "baz",
+						OSTree: distro.OSTreeImageOptions{
+							URL:           "foo",
+							ImageRef:      "bar",
+							FetchChecksum: "baz",
```

Comment:
I didn't bother changing this.  We could do it at a later stage since it doesn't really affect much, it's just a cleaner test if it doesn't add unnecessary options.

Reply:
Resolving this.  Can clean up another time.